### PR TITLE
Revert "ENG-2677 Fix content config templates fetching"

### PR DESCRIPTION
**Pull request by user-54dbb57a**
Reverts entando/entando-cms#427

Hotfix was created on the wrong release branch

### CODE REVIEW

**Review by raxkaynan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-cms/pull/429#pullrequestreview-786580136" title="Approved">raxkaynan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-cms/blob/release/0.0.278/OWNERS)~~ [raxkaynan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
